addons/web/static/src/js/libs/underscore.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
I would put it in a `odoo.define('web.underscore.extensions', function () {` like other extensions.

Reply:
Missing `'use strict';`